collections/Lyell/MS_Lyell_50.xml

```diff
@@ -8,17 +8,11 @@
             <title>MS. Lyell 50</title>
             <title type="collection">MSS. Lyell</title>
             <respStmt>
-               <resp>Summary description</resp>
-               <persName>Elizabeth Solopova</persName><persName>Matthew Holford</persName>
+               <resp>Description</resp>
```

Comment:
I normally separate this into two sections (which can be separately dated) - `<resp>Cataloguer</resp>` and `<resp>Encoding</resp>`